flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartition.java

```diff
@@ -317,7 +317,6 @@ public ResultSubpartitionView createSubpartitionView(
 				availabilityListener,
 				resultFile);
 			readers.add(reader);
-			availabilityListener.notifyDataAvailable();
```

Comment:
Could you please also add the component name to the commit message? Also `This closes #13991.` should probably appear in the last commit.

Reply:
Fixed.